pages/give-devices.js

```diff
@@ -108,11 +108,11 @@ export default function GiveDevicesPage() {
       </Head>
       <Navbar />
       <main>
-        <div className="m-auto w-3/4 p-10 my-10 bg-green-300 border border-gray-300">
-          <div className="mt-10 sm:mt-0">
+        <div className="p-12" style={{ marginTop: '-.5rem' }}>
```

Comment:
Any reason why `-mt-2` isn't being using instead of `marginType`? Reference: https://tailwindcss.com/docs/margin